src/packages/recompose/__tests__/types/test_withHandlers.js

```diff
@@ -13,7 +13,6 @@ type EnhancedCompProps = {
 }
 
 const enhancer: HOC<*, EnhancedCompProps> = compose(
-  // $FlowFixMe[incompatible-use]
```

Comment:
Removing the function branch made these `$FlowFixMe`s in the type tests unnecessary.